src/pydocstyle/checker.py

```diff
@@ -91,42 +108,55 @@ class ConventionChecker:
     # " random         : test" where random will be captured as the param
     # "  random_t (Test) : test  " where random_t will be captured as the param
     GOOGLE_ARGS_REGEX = re(
-                        # Matches anything that fulfills all the following conditions:
```

Comment:
Can you move each comment to the line above the regex it describes? This hurts readability IMO.

Reply:
Fixed :)